OpenThaiGPT/openthaigpt-pretraining
# feat(model): add LLaMa Tokenizer Supported Thai Token

merge LLaMa Tokenizer with Thai Tokenizer by extent vocabulary following [working example](https://github.com/ymcui/Chinese-LLaMA-Alpaca/blob/main/README_EN.md) 


- To test merged method
1. already have LLaMa Tokenizer and Thai Tokenizer
1.  run merge_tokenizer.py
``` python3 ./openthaigpt-pretraining/src/model/openthaigpt_pretraining_model/llama_thai_tokenizer/merge_tokenizer.py```
2. merge tokenizer will not save on if you want to save you must uncomment line in merge_tokenizer.py
- To test merged tokenizer
1.  run llama_thai_token_test.py and inference time check
```python3 ./openthaigpt-pretraining/src/model/scripts/llama_thai_token_test.py```


Close #77
